symbiflow.py

```diff
@@ -505,9 +505,11 @@ def run(self):
 
                 chip = self.family + self.device
 
+                conda_dir = os.getenv("CONDA_PREFIX", None)
```

Comment:
In general, don't depend on conda directly.  FYI @HackerFoo

Reply:
Yes, code should be independent of the package management system used if possible.